dnf-behave-tests/features/plugins-core/versionlock-lock.feature

```diff
@@ -234,3 +234,40 @@ Scenario: Check-update command does not report updates filtered out by the versi
     """
     <REPOSYNC>
     """
+
+
+# PackageA-Obsoleter obsoletes PackageB < 3.0
```

Comment:
Is this comment perhaps leftover from some other version of the test? Or how is PackageA-Obsoleter relevant here?

Reply:
Yes, it's a typo. Fixed.

Reply:
Thanks!